src/platform/static-data/SubmitSignInForm.jsx

```diff
@@ -22,20 +19,11 @@ export default function SubmitSignInForm({ startSentence }) {
     </a>
   );
 
-  const ttyLink = (
-    <a
-      aria-label={formatAriaLabel(ttyNumber, true)}
-      href={createHref(ttyNumber)}
-    >
-      {formatPhoneNumber(ttyNumber, true)}
-    </a>
-  );
-
   return (
     <span>
-      {startSentence ? 'Call' : 'call'} our MyVA411 main information line for
-      help at {helpDeskLink} ({ttyLink}
-      ).
+      {startSentence ? 'Call' : 'call'} our{' '}
+      <va-telephone contact="8662793677" /> technical support line for help at{' '}
```

Comment:
The "va-telephone" should just have text of "VA.gov technical support line..." and the VA.gov part is not a link. Let me know if you need to sync real quick.